File: Subastas-app/subastas-client/src/dto/auth/PreRegisterRequestDTO.ts
export interface PreRegisterRequestDTO {
  nombre: string;
  apellido: string;
  documento: string;
  mail: string;

  frenteDNIUrl: string;
  dorsoDNIUrl: string;

  domicilio: {
    pais: string;
    provincia: string;
    ciudad: string;
    cp: string;
    direccion: string;
  };
}

File: Subastas-app/subastas-api/src/main/java/com/subastas/subastas_api/DTO/auth/PreRegisterRequestDTO.java
package com.subastas.subastas_api.DTO.auth;

import com.subastas.subastas_api.model.Domicilio;

public class PreRegisterRequestDTO {

    private String nombre;
    private String apellido;
    private String documento;
    private String mail;
    private String frenteDNIUrl;
    private String dorsoDNIUrl;
    private Domicilio domicilio;

    public String getNombre() {
        return nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public String getDocumento() {
        return documento;
    }

    public String getMail() {
        return mail;
    }

    public String getFrenteDNIUrl() {
        return frenteDNIUrl;
    }

    public String getDorsoDNIUrl() {
        return dorsoDNIUrl;
    }

    public Domicilio getDomicilio() {
        return domicilio;
    }
}